Lib/Salesforce/SalesforceAuthManager.php

```diff
@@ -27,6 +27,9 @@ public function __construct() {
 			'clientSecret'      => Configure::read('Salesforce.client_secret'),
 			'redirectUri'       => $this->_getCallbackUrl(MediaPlatform::SALESFORCE)
 		]);
+		if (Configure::read('DEO.live') !== true) {
+			$this->salesforce->setDomain('https://test.salesforce.com');
```

Comment:
De salesforce url (of environment) moet een configuratie zijn in authmanager, niet configuratie van DEO gebruiken.